README.md

```diff
@@ -15,13 +15,13 @@ $ npm install add-dependencies [-g]
 Go to a directory with the target `package.json` and run:
```

Comment:
README should be rewritten more comprehensively now. There's no need to go to a specific directory since we make what we make.

Reply:
addressed